brillig_vm/src/lib.rs

```diff
@@ -43,22 +43,28 @@ pub struct ForeignCallResult {
 }
 
 #[derive(Debug, PartialEq, Eq, Clone)]
+// VM encapsulates the state of the Brillig VM during execution.
 pub struct VM {
+    // Register storage
     registers: Registers,
+    // Instruction pointer
     program_counter: usize,
-    /// A counter maintained throughout a Brillig process that determines
-    /// whether the caller has resolved the results of a [foreign call][Opcode::ForeignCall].
+    // Tracks foreign calls
     foreign_call_counter: usize,
-    /// Represents the outputs of all foreign calls during a Brillig process
-    /// List is appended onto by the caller upon reaching a [VMStatus::ForeignCallWait]
+    // Holds foreign call results
     foreign_call_results: Vec<ForeignCallResult>,
```

Comment:
Are these removed comments no longer true? I think they are helpful.

Reply:
I am curious about this removal as well

Reply:
Just an accident